functions/go/create-setters/metadata.yaml

```diff
@@ -2,9 +2,9 @@ image: gcr.io/kpt-fn/create-setters
 description: Create setter values on resource fields. Setters serve as parameters for template-free setting of field values.
```

Comment:
This should be `Add setter comments to matching resource fields.` That will be short single sentence consistent with other functions.